src/background.js

```diff
@@ -8,7 +8,7 @@
 
 // Import the WebExtensions polyfill, for cross-browser compatibility.
 // Note that Rally and WebScience currently only support Firefox.
-import "webextension-polyfill";
+// import { browser } from "webextension-polyfill";
```

Comment:
Things are starting to get complicated with the polyfill, since some of our dependencies expect it to be automatically injected and some explicitly imported. Leaving it commented out for now and just including the polyfill when we load things, I'll tackle cleaning this up as a separate issue.